lib/route_translator.rb

```diff
@@ -5,6 +5,7 @@
 require File.expand_path('../route_translator/extensions', __FILE__)
 require File.expand_path('../route_translator/translator', __FILE__)
 require File.expand_path('../route_translator/host', __FILE__)
+require_relative 'route_translator/host_path_consistency_lambdas'
```

Comment:
I thought this was ok since this gem depends on rails 5, and rails 5 depends on ruby 2.2